index.js

```diff
@@ -433,6 +433,24 @@ module.exports = {
       return file.indexOf('.js') > -1;
     });
 
+    // Allows for custom paths to be searched
+    let amdModulePaths = this.app.options.amd.amdModulePaths || [];
+    amdModulePaths.forEach(function(amdModulePath) {
+      // Replace all slashes with platform specific separator
+      var replacedPath = amdModulePath.replace(/\/|\\/g, path.sep);
```

Comment:
I'm pretty sure you can delete this line and let the `path` module do the work for you